sdk/client/clientlist.go

```diff
@@ -98,3 +98,7 @@ func (nsmc *NsmClientList) Connect(name, mechanism, description string) error {
 	}
 	return nil
 }
+
+func (nsmc *NsmClientList) GetConnections() []*connection.Connection {
```

Comment:
That is interesting. shall we just extend Connect to return this list alongside the error?

Reply:
Your proposal makes sense, since sdk.client.Connect() returns a Connection, sdk.clientList.Connect() should return a slice of Connection.

I'll update my patch this way.